testing/pypy3/alpine.patch

```diff
@@ -0,0 +1,187 @@
+diff -r 7f9407387dbf lib_pypy/_tkinter/tklib_build.py
```

Comment:
Where is this patch from? Have you written it or copied from somewhere? What exactly does it fixes? It seems that it fixes multiple unrelated things.

Please

1. split this patch to multiple patches, each focused on single thing (e.g. PaX, tcl/tk, …),
2. add proper patch header with origin of the patch and brief explanation what does it do and why (if it’s not obvious from the patch itself),
3. fix mixed tabs and spaces.